src/language-js/utils/index.js

```diff
@@ -329,11 +329,13 @@ function isSimpleType(node) {
  * @returns {boolean}
  */
 function isUnitTestSetUp(node) {
-  const unitTestSetUpRe = /^(?:before|after)(?:Each|All)$/;
   return (
     node.callee.type === "Identifier" &&
     node.arguments.length === 1 &&
```

Comment:
`node.arguments` already checked in `isTestCall`, let's remove it, since we are not checking `node.type` here either.